packages/create-discord-bot/template/TypeScript/src/util/deploy.ts

```diff
@@ -0,0 +1,15 @@
+import process from 'node:process';
+import { URL } from 'node:url';
+import { API } from '@discordjs/core/http-only';
+import { REST } from 'discord.js';
+import { loadCommands } from './loaders.js';
+
+const commands = await loadCommands(new URL('commands/', import.meta.url));
+const commandData = [...commands.values()].map((command) => command.data);
```

Comment:
why not just .map?

Reply:
`values` returns an iterator not an array. This will be possible in the future though: https://github.com/tc39/proposal-iterator-helpers

Reply:
but commands is an array

Reply:
Isn't it a `Map`?

Reply:
Yes, it’s a `Map`, see https://github.com/discordjs/discord.js/blob/26ee2b36a87bb04c15a8b18954694d06b6e874e9/packages/create-discord-bot/template/TypeScript/src/util/loaders.ts#LL67C1-L67C1

Reply:
Yeahh I misreaded